packages/react/src/components/Forms/Select/mono-searchable.tsx

```diff
@@ -39,6 +47,33 @@ export const SelectMonoSearchable = forwardRef<SelectHandle, SubProps>(
     const [labelAsPlaceholder, setLabelAsPlaceholder] = useState(
       !downshiftReturn.selectedItem,
     );
+
+    const computeOptionsToDisplay = async (
+      context: ContextCallbackFetchOptions,
+    ): Promise<void> => {
+      const arrayOptions = await fetchOptions(context);
+
+      setOptionsToDisplay(arrayOptions);
+    };
+
+    const fetchOptions = async (
+      context: ContextCallbackFetchOptions,
+    ): Promise<Option[]> => (props.options as CallbackFetchOptions)(context);
+
+    const computeDefaultOptionToSelect = async (defaultValue: string) => {
```

Comment:
To prevent a refactoring if we update the Option type to allow another type than string as option value you should type `defaultValue` like that IMO `Option['value']`